py/client/pydeephaven/session.py

```diff
@@ -330,7 +333,7 @@ def is_alive(self) -> bool:
                 return True
 
             try:
-                self.session_service.refresh_token()
+                self._flight_client.authenticate(self._auth_handler)
```

Comment:
I don't understand the logic behind this line change.

Reply:
It was an overlooked bug. `self.session_service` doesn't have the `refresh_token` method anymore after the switch to flight based auth.